wifiutils/src/main/java/com/thanosfisherman/wifiutils/WifiConnectorBuilder.java

```diff
@@ -27,6 +27,9 @@
         @NonNull
         WifiConnectorBuilder scanWifi(@Nullable ScanResultsListener scanResultsListener);
 
+        @NonNull
+        WifiSuccessListener connectWith(@NonNull String ssid);
```

Comment:
IDK if that method is required, maybe existing `connectWith` methods are enough

Reply:
Let's keep it for now :)